### PR TITLE
Tooltip for post logout redirect uri needs updating for Keycloak issue 14884

### LINKED ISSUE #3698: Tooltip for post logout redirect uri needs updating for Keycloak issue 14884
### Describe the bug

Tooltip needs to be updated to reflect changes made for Keycloak issue 14884. Need to describe the placeholder '-'.

### Version

_No response_

### Environment

_No response_

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by douglaspalmer**
Closes #3698

### CODE REVIEW

**Comment by douglaspalmer** [comment]
Should be merged after https://github.com/keycloak/keycloak/pull/15298

File: apps/admin-ui/public/resources/en/clients-help.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
...
-  "validPostLogoutRedirectURIs": "Valid URI pattern a browser can redirect to after a successful logout. A value of '+' will use the list of valid redirect uris. Simple wildcards are allowed such as 'http://example.com/*'. Relative path can be specified too such as /my/relative/path/*. Relative paths are relative to the client root URL, or if none is specified the auth server root URL is used. For SAML, you must set valid URI patterns if you are relying on the consumer service URL embedded with the login request.",
+  "validPostLogoutRedirectURIs": "Valid URI pattern a browser can redirect to after a successful logout. A value of '+' will use the list of valid redirect uris. A value of '-' will use an empty list of valid redirect uris. Simple wildcards are allowed such as 'http://example.com/*'. Relative path can be specified too such as /my/relative/path/*. Relative paths are relative to the client root URL, or if none is specified the auth server root URL is used. For SAML, you must set valid URI patterns if you are relying on the consumer service URL embedded with the login request.",
```
Comment (sschu) [inline apps/admin-ui/public/resources/en/clients-help.json:16]: See my comment here: https://github.com/keycloak/keycloak/pull/15298#discussion_r1012072751

**Review by ssilvert — CHANGES_REQUESTED** [review]
We have a problem with the other languages because they aren't getting updated.  I'll ping the rest of the UI team to see if they have any ideas about what we should do in this case.

**Comment by ssilvert** [comment]
@douglaspalmer I think what we need to do for now is just to remove that key for the other languages.  Then it will default to English and the community will send us the proper translations later.

**Comment by sschu** [comment]
I could provide German translations once we agreed on a text.

**Comment by ssilvert** [comment]
@jonkoops I agree.  Please create an issue for it.

@sschu @douglaspalmer Since the text is not agreed upon I'm going to close this PR.  I'd prefer to have a new PR when the dependent PR is merged and the text is finalized.  At that time, we can provide translations for those languages where we are proficient.  Also, @douglaspalmer, when you create the new PR, please remove the keys for those languages that don't have a translation.

**Comment by jonkoops** [comment]
@ssilvert Done, see #3712. Feel free to add to the requirements.

**Pull request closed without merging**